unit_test/entities/test_entities.py

```diff
@@ -24,9 +24,10 @@ class HeatNetworkTest(unittest.TestCase):
     def test_heat_network(self) -> None:
         """Generic/template test for Heatnetwork."""
         # Arrange
-
+        assets = []
+        junctions = []
         # Act
-        result = HeatNetwork()
+        result = HeatNetwork(assets, junctions)
```

Comment:
What does the HeatNetwork(assets, junctions) call do when the assets and junctions are an empty list? Do we want to be able to create an empty HeatNetwork entity? 

If so, maybe we should add a default empty list already to the init of HeatNetwork? 

@MichielTukker

Reply:
If we implement more in heatnetwork class then this can also be covered. I already made an issue for extending the heat network class #42